src/SayMore/ProjectContext.cs

```diff
@@ -46,7 +46,7 @@ public ProjectContext(string projectSettingsPath, IContainer parentContainer)
 
 			Project = _scope.Resolve<Func<string, Project>>()(projectSettingsPath);
 
-			SetContributorsListToSession(Project.SessionsFolder);
+			SetContributorsListAndMigrateAccessProtocolToSession(Project.SessionsFolder, Project.AccessProtocol);
```

Comment:
I realize that this is just a rename, not a new call (as it was apparently added a few weeks ago), but I'm not very sure I would expect a "ProjectContext" constructor to call a method that alters the saved data for another object. Was this design thought out, or was this method call just slapped in here for convenience because it happened to be called at the "right time"?